src/main/java/com/contentful/java/cda/CDAClient.java

```diff
@@ -228,7 +266,13 @@ public CDASpace fetchSpace() {
     return Observable.just(contentType);
   }
 
-  /** Returns a {@link CDAClient} builder. */
+  String createUserAgent() {
+    return String.format("contentful.java/%s", Util.getProperty("version.name"));
```

Comment:
Might be interesting to add some more information here, e.g. the iOS SDK also logs the OS version.